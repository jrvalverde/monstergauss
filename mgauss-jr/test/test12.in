*MOL BA=G,RHF,CO=M
*PRT IGC=Y
CARBON
 0 3
 6

3330E
 1
     1S  1.0
153.396             0.0707666
23.0596             0.395356
4.92985             0.663169
     2S  1.0
4.39495             -0.0926218
0.658157            0.340674
0.198374            0.743772
     2P  1.0
4.23235                                 0.110281
0.861673                                0.462034
0.201326                                 0.627665
****

FA
23
22
20
*MOL BA=G,RHF
NITROGEN
 0 4
 7

3330E
 1
     1S  1.0
210.633             0.0705931
31.7445             0.395908
6.82708             0.662113
     2S  1.0
6.59787             -0.0890379
0.879820            0.379306
0.269261            0.704726
     2P  1.0
6.11362                                 0.115967
1.25946                                 0.469361
0.291612                                0.618892
****

FA
23


*MOL BA=G,RHF
OXYGEN
 0 3
 8

3330E
 1
     1S  1.0
277.555             0.0703080
41.8923             0.395853
9.04437             0.661894
     2S  1.0
9.28383             -0.0875258
1.14812             0.407671
0.353831            0.676432
     2P  1.0
8.09648                                 0.123326
1.67764                                 0.475101
0.374101                                0.615173
****

FA
23
23
21
