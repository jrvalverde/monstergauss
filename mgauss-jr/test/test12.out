 WELCOME TO U OF T MONSTERGAUSS                                                                         UNX VERSION: 13 AUG 1991
 ******************************
0CONTROL CARDS:

 *MOL BA=G,RHF,CO=M                                                      
 *PRT IGC=Y                                                              

0SEQUENCE    GROUP     MG OPTIONS         GROUP OPTIONS (USER CONTROLLABLE)                  LINKS TO BE CALLED          JUMP

     1         2      000007012100 000000000000000000000000000000000000000000000000   01000000000000000000000000000000     0
     2         3      000007012100 004000010020000000000000000000000000000000000000   11000100000000000000000000000000     0
     3         4      000007012100 100000010000000010000000000000000000000000000000   11000000000000000000000000000000     0
     4         5      000007012100 170000000000000010000000000000000000000000000000   00100000000000000000000000000000     0
     5         6      000007012100 000000000000000000001000000000000000000000000000   10000000000000000000000000000000     0
0OPTION NUMBERS:      000000000111 111111122222222223333333333444444444455555555556   66666666677777777778888888888999
                      123456789012 345678901234567890123456789012345678901234567890   12345678901234567890123456789012

0CONVERSION FACTORS:  1 BOHR =  .52917706 ANGSTROMS
0                     1 HARTREE/BOHR = 8.2388584 MDYNE

                      1 HARTREE/ANGSTROM**2 = 4.3598149 MDYNE/ANGSTROM

                      1 ELECTRON*BOHR = 2.5417655 DEBYE

                      1 HARTREE/PARTICLE = 627.50959 KCAL/MOL = 2625.5001 KJ/MOL = 27.211611 EV/PARTICLE = 219474.67 CM(-1)/PARTICLE

                      1 ATOMIC UNIT OF ELECTRIC FIELD = 5.1422506D+09 VOLTS/CM
1CARBON                                                                  
 ------------------------------------------------------------------------
0CHARGE  =   0          MULTIPLICITY  =  3

0FREE FORMAT Z MATRIX:

  6                                                                      
                                                                         
0NO VARIABLES USED - SWITCHING TO STANDARD Z MATRIX FORMAT FOR OUTPUT
1
0---------------------------------------------------------------------------------------------------
                                             Z MATRIX
0  I   AN   Z1      BL                  Z2     ALPHA                Z3      BETA                Z4
 ---------------------------------------------------------------------------------------------------
   1    6
 ---------------------------------------------------------------------------------------------------


 ------------------------------------------------------------------------------------------------------------------------------------
                    COORDINATES IN ANGSTROMS                               COORDINATES IN BOHR                    ATOMIC ISOTOPE/MASS
   I   AN        X             Y             Z                        X             Y             Z               MASS#    MASS (AMU)
 ------------------------------------------------------------------------------------------------------------------------------------
   1    6     .0000000      .0000000      .0000000                 .0000000      .0000000      .0000000             12    12.00000000
 ------------------------------------------------------------------------------------------------------------------------------------

0PROJECTING FROM STO-3G BASIS (5D/7F)

0BASIS SET TYPE: EXTERNAL GENERAL     ( 5D/ 7F)
 POLARIZATION  : UNKNOWN                                                                              
 DIFFUSE FUNCS : UNKNOWN                                                                              
1BASIS SET FOR THIS CALCULATION (EXPONENTS ARE SCALED)

 **********************************************************************************************************************************
               ATOMIC ORBITAL                                     GAUSSIAN FUNCTIONS
 **********************************************************************************************************************************
      ATOM       NO.    TYPE   SCALE FACTOR    EXPONENT        S COEF         P COEF         D COEF         F COEF
 **********************************************************************************************************************************
       CARBON
                    1    1S       1.00000
                                             1.5339600D+02  7.0766600D-02   .0000000D+00   .0000000D+00   .0000000D+00
                                             2.3059600D+01  3.9535600D-01   .0000000D+00   .0000000D+00   .0000000D+00
                                             4.9298500D+00  6.6316900D-01   .0000000D+00   .0000000D+00   .0000000D+00
                    2    2S       1.00000
                                             4.3949500D+00 -9.2621800D-02   .0000000D+00   .0000000D+00   .0000000D+00
                                             6.5815700D-01  3.4067400D-01   .0000000D+00   .0000000D+00   .0000000D+00
                                             1.9837400D-01  7.4377200D-01   .0000000D+00   .0000000D+00   .0000000D+00
                3-  5    2P       1.00000
                                             4.2323500D+00   .0000000D+00  1.1028100D-01   .0000000D+00   .0000000D+00
                                             8.6167300D-01   .0000000D+00  4.6203400D-01   .0000000D+00   .0000000D+00
                                             2.0132600D-01   .0000000D+00  6.2766500D-01   .0000000D+00   .0000000D+00
 **********************************************************************************************************************************

0*** THE S=P EXPONENT CONSTRAINT HAS BEEN REMOVED
0RYSSET ROOT STATISTICS:     INTERPOLATED:       54     CALCULATED:        0     ASSYMPTOTIC APPROXIMATION:        0
0STANDARD CUTOFFS SELECTED IN SHELL.


         33 TWO ELECTRON INTEGRALS WRITTEN OUT ON UNIT  9 WITH INTEGRAL CUTOFF  1.00000D-10
1RHF OPEN SHELL SPECIFICATION
0GROUP     : FA   
 DEGENERACY: 23000
 ALPHA OCC : 22000
 BETA OCC  : 20000

0ORBITAL GROUP :       F       D       S       A       V
0NUMBER OF MO  :       2       0       0       3       0

0LEVEL    DEGENERACY    DOUBLE OCC    SINGLE ALPHA    SINGLE BETA    CONFIGURATIONS    SAMPLE CONFIGURATION
0  1           3             0              2               0               3          A   A       
+                                                                                      __  __  __  

0COULOMB COUPLING COEFFICIENTS   A(I,J)
0  MO          OCCA      OCCB         CLOSED   OPEN 1   OPEN 2   OPEN 3   
0CLOSED       2.0000    2.0000        2.0000    .6667    .6667    .6667
 OPEN 1        .6667     .0000         .6667    .0000    .1667    .1667
 OPEN 2        .6667     .0000         .6667    .1667    .0000    .1667
 OPEN 3        .6667     .0000         .6667    .1667    .1667    .0000

0EXCHANGE COUPLING COEFFICIENTS   B(I,J)
0  MO          OCCA      OCCB         CLOSED   OPEN 1   OPEN 2   OPEN 3   
0CLOSED       2.0000    2.0000        1.0000    .3333    .3333    .3333
 OPEN 1        .6667     .0000         .3333    .0000    .1667    .1667
 OPEN 2        .6667     .0000         .3333    .1667    .0000    .1667
 OPEN 3        .6667     .0000         .3333    .1667    .1667    .0000

0PH INITIAL GUESS ... SCALE FACTOR (K/2) =  .8750
 SMALLEST EIGENVALUE OF OVERLAP IS  7.51638D-01
 SMALLEST EIGENVALUE OF OVERLAP IS  7.99716D-01
 COEFFICIENTS PROJECTED TO CURRENT BASIS
1INITIAL GUESS COEFFICIENTS         4 ALPHA SPIN ELECTRONS,    2 BETA SPIN ELECTRONS

                           1          2          3          4          5
 
    EIGENVALUES---   -11.52004    -.90555    -.37000    -.37000    -.37000
 
   1  1   C   1S        .99723    -.21754     .00000     .00000     .00000
   2  1   C   2S        .01340    1.02059     .00000     .00000     .00000
   3  1   C   2PX       .00000     .00000    1.00000     .00000     .00000
   4  1   C   2PY       .00000     .00000     .00000    1.00000     .00000
   5  1   C   2PZ       .00000     .00000     .00000     .00000    1.00000
 
 
 THESE MO COEFFICIENTS HAVE BEEN WRITTEN TO FILE 25, RE-ORDERED BY GROUPS
 THESE MO COEFFICIENTS HAVE BEEN WRITTEN TO FILE 27, RE-ORDERED BY GROUPS
1RHF OPEN SHELL SCF               NUCLEAR REPULSION ENERGY IS       .000000000  HARTREES
0CONVERGENCE ON DENSITY MATRIX REQUIRED TO EXIT IS  1.0000D-06,  INITIAL SCALE FACTOR (SCALST) IS  .80000
 SCALE FACTOR CUT RATIO (FACTDN) IS  .66667,  SCALE FACTOR INCREASE RATIO (FACTUP) IS 1.20000
 SCALE FACTOR LOWER LIMIT (SLIMDN) IS  .00050,  SCALE FACTOR UPPER LIMIT (SLIMUP) IS  .80000
 SWITCH TO INCREASE MODE AT D CONVERGENCE  2.0000D-03 (SCRIT, RHF=INCR MODE ONLY).
0 CYCLE                   ENERGY                    SCF CONVERGENCE         CI CONVERGENCE         EXTRAPOLATION     SCALE      RESET
0 CI SCF       ELECTRONIC           TOTAL          COEFF      DENSITY      COEFF      DENSITY

   1   0      -37.448434407      -37.448434407             
   1   1      -37.448434407      -37.448434407                                       .0000D+00                     8.0000D-01    YES 
0AT TERMINATION TOTAL ENERGY IS      -37.448434  HARTREES          ORBITAL CONVERGENCE IS  .00000D+00

0S = 1.000 ( 1.000 EXPECTED),   S(S+1) =  2.0000 (  2.0000 EXPECTED)
1MOLECULAR ORBITALS         4 ALPHA SPIN ELECTRONS,   2 BETA SPIN ELECTRONS

                           1          2          3          4          5
 
    EIGENVALUES---   -11.22574    -.69631    -.14027    -.14027    -.14027
 
   1  1   C   1S        .99271    -.23732     .00000     .00000     .00000
   2  1   C   2S        .03368    1.02013     .00000     .00000     .00000
   3  1   C   2PX       .00000     .00000    1.00000     .00000     .00000
   4  1   C   2PY       .00000     .00000     .00000    1.00000     .00000
   5  1   C   2PZ       .00000     .00000     .00000     .00000    1.00000
 
 
 THE FIRST   2 MO ARE DOUBLY OCCUPIED FROZEN CORE ORBITALS
0OPEN SHELL ORBITAL OCCUPANCIES:    MO     OCCA     OCCB
                                     3     .6667    .0000
                                     4     .6667    .0000
                                     5     .6667    .0000
1MULLIKEN POPULATION MATRIX CONDENSED TO ATOMS (ALPHA SPIN)     (DOUBLE THE OFF-DIAGONAL ELEMENTS FOR 'BOND ORDERS')

                           1
 
         1   C         4.00000
 
1MULLIKEN POPULATION MATRIX CONDENSED TO ATOMS (BETA SPIN)      (DOUBLE THE OFF-DIAGONAL ELEMENTS FOR 'BOND ORDERS')

                           1
 
         1   C         2.00000
 
1MULLIKEN POPULATION MATRIX CONDENSED TO ATOMS (TOTAL)          (DOUBLE THE OFF-DIAGONAL ELEMENTS FOR 'BOND ORDERS')

                           1
 
         1   C         6.00000
 
0ATOMIC CHARGES (ALPHA(1), BETA(2)), ATOMIC SPIN DENSITIES(3), ATOMIC CHARGES (TOTAL(4), NET(5))

                           1          2          3          4          5
 
         1   C         4.00000    2.00000    2.00000    6.00000     .00000
 
1BOND ORDER AND VALENCE INDICES (I. MAYER, I.J.Q.C., 29, 477-483 (1986))


          BOND ORDER MATRIX

                           1
 
         1   C          .00000
 
0                      VALENCE  FREE VALENCE

                           1          2
 
         1   C         2.66667    2.66667
 

0NORMAL TERMINATION, TOTAL CPU TIME FOR THIS CALCULATION WAS      .003 MINUTES.
1WELCOME TO U OF T MONSTERGAUSS                                                                         UNX VERSION: 13 AUG 1991
 ******************************
0CONTROL CARDS:

 *MOL BA=G,RHF                                                           

0SEQUENCE    GROUP     MG OPTIONS         GROUP OPTIONS (USER CONTROLLABLE)                  LINKS TO BE CALLED          JUMP

     1         2      000007012100 000000000000000000000000000000000000000000000000   01000000000000000000000000000000     0
     2         3      000007012100 004000010020000000000000000000000000000000000000   11000100000000000000000000000000     0
     3         4      000007012100 100000000000000010000000000000000000000000000000   11000000000000000000000000000000     0
     4         5      000007012100 100000000000000010000000000000000000000000000000   00100000000000000000000000000000     0
     5         6      000007012100 000000000000000000001000000000000000000000000000   10000000000000000000000000000000     0
0OPTION NUMBERS:      000000000111 111111122222222223333333333444444444455555555556   66666666677777777778888888888999
                      123456789012 345678901234567890123456789012345678901234567890   12345678901234567890123456789012

0CONVERSION FACTORS:  1 BOHR =  .52917706 ANGSTROMS
0                     1 HARTREE/BOHR = 8.2388584 MDYNE

                      1 HARTREE/ANGSTROM**2 = 4.3598149 MDYNE/ANGSTROM

                      1 ELECTRON*BOHR = 2.5417655 DEBYE

                      1 HARTREE/PARTICLE = 627.50959 KCAL/MOL = 2625.5001 KJ/MOL = 27.211611 EV/PARTICLE = 219474.67 CM(-1)/PARTICLE

                      1 ATOMIC UNIT OF ELECTRIC FIELD = 5.1422506D+09 VOLTS/CM
1NITROGEN                                                                
 ------------------------------------------------------------------------
0CHARGE  =   0          MULTIPLICITY  =  4

0FREE FORMAT Z MATRIX:

  7                                                                      
                                                                         
0NO VARIABLES USED - SWITCHING TO STANDARD Z MATRIX FORMAT FOR OUTPUT
1
0---------------------------------------------------------------------------------------------------
                                             Z MATRIX
0  I   AN   Z1      BL                  Z2     ALPHA                Z3      BETA                Z4
 ---------------------------------------------------------------------------------------------------
   1    7
 ---------------------------------------------------------------------------------------------------


 ------------------------------------------------------------------------------------------------------------------------------------
                    COORDINATES IN ANGSTROMS                               COORDINATES IN BOHR                    ATOMIC ISOTOPE/MASS
   I   AN        X             Y             Z                        X             Y             Z               MASS#    MASS (AMU)
 ------------------------------------------------------------------------------------------------------------------------------------
   1    7     .0000000      .0000000      .0000000                 .0000000      .0000000      .0000000             14    14.00307401
 ------------------------------------------------------------------------------------------------------------------------------------

0PROJECTING FROM STO-3G BASIS (5D/7F)

0BASIS SET TYPE: EXTERNAL GENERAL     ( 5D/ 7F)
 POLARIZATION  : UNKNOWN                                                                              
 DIFFUSE FUNCS : UNKNOWN                                                                              
1BASIS SET FOR THIS CALCULATION (EXPONENTS ARE SCALED)

 **********************************************************************************************************************************
               ATOMIC ORBITAL                                     GAUSSIAN FUNCTIONS
 **********************************************************************************************************************************
      ATOM       NO.    TYPE   SCALE FACTOR    EXPONENT        S COEF         P COEF         D COEF         F COEF
 **********************************************************************************************************************************
     NITROGEN
                    1    1S       1.00000
                                             2.1063300D+02  7.0593100D-02   .0000000D+00   .0000000D+00   .0000000D+00
                                             3.1744500D+01  3.9590800D-01   .0000000D+00   .0000000D+00   .0000000D+00
                                             6.8270800D+00  6.6211300D-01   .0000000D+00   .0000000D+00   .0000000D+00
                    2    2S       1.00000
                                             6.5978700D+00 -8.9037900D-02   .0000000D+00   .0000000D+00   .0000000D+00
                                             8.7982000D-01  3.7930600D-01   .0000000D+00   .0000000D+00   .0000000D+00
                                             2.6926100D-01  7.0472600D-01   .0000000D+00   .0000000D+00   .0000000D+00
                3-  5    2P       1.00000
                                             6.1136200D+00   .0000000D+00  1.1596700D-01   .0000000D+00   .0000000D+00
                                             1.2594600D+00   .0000000D+00  4.6936100D-01   .0000000D+00   .0000000D+00
                                             2.9161200D-01   .0000000D+00  6.1889200D-01   .0000000D+00   .0000000D+00
 **********************************************************************************************************************************

0*** THE S=P EXPONENT CONSTRAINT HAS BEEN REMOVED
0RYSSET ROOT STATISTICS:     INTERPOLATED:       54     CALCULATED:        0     ASSYMPTOTIC APPROXIMATION:        0
0STANDARD CUTOFFS SELECTED IN SHELL.


         33 TWO ELECTRON INTEGRALS WRITTEN OUT ON UNIT  9 WITH INTEGRAL CUTOFF  1.00000D-10
1RHF OPEN SHELL SPECIFICATION
0GROUP     : FA   
 DEGENERACY: 23000
 ALPHA OCC : 00000
 BETA OCC  : 00000

0ORBITAL GROUP :       F       D       S       A       V
0NUMBER OF MO  :       2       0       0       3       0

0LEVEL    DEGENERACY    DOUBLE OCC    SINGLE ALPHA    SINGLE BETA    CONFIGURATIONS    SAMPLE CONFIGURATION
0  1           3             0              3               0               1          A   A   A   
+                                                                                      __  __  __  

0COULOMB COUPLING COEFFICIENTS   A(I,J)
0  MO          OCCA      OCCB         CLOSED   OPEN 1   OPEN 2   OPEN 3   
0CLOSED       2.0000    2.0000        2.0000   1.0000   1.0000   1.0000
 OPEN 1       1.0000     .0000        1.0000    .0000    .5000    .5000
 OPEN 2       1.0000     .0000        1.0000    .5000    .0000    .5000
 OPEN 3       1.0000     .0000        1.0000    .5000    .5000    .0000

0EXCHANGE COUPLING COEFFICIENTS   B(I,J)
0  MO          OCCA      OCCB         CLOSED   OPEN 1   OPEN 2   OPEN 3   
0CLOSED       2.0000    2.0000        1.0000    .5000    .5000    .5000
 OPEN 1       1.0000     .0000         .5000    .0000    .5000    .5000
 OPEN 2       1.0000     .0000         .5000    .5000    .0000    .5000
 OPEN 3       1.0000     .0000         .5000    .5000    .5000    .0000

0PH INITIAL GUESS ... SCALE FACTOR (K/2) =  .8750
 SMALLEST EIGENVALUE OF OVERLAP IS  7.64962D-01
 SMALLEST EIGENVALUE OF OVERLAP IS  7.94064D-01
 COEFFICIENTS PROJECTED TO CURRENT BASIS
 
 THESE MO COEFFICIENTS HAVE BEEN WRITTEN TO FILE 25, RE-ORDERED BY GROUPS
 THESE MO COEFFICIENTS HAVE BEEN WRITTEN TO FILE 27, RE-ORDERED BY GROUPS
1RHF OPEN SHELL SCF               NUCLEAR REPULSION ENERGY IS       .000000000  HARTREES
0CONVERGENCE ON DENSITY MATRIX REQUIRED TO EXIT IS  1.0000D-06,  INITIAL SCALE FACTOR (SCALST) IS  .80000
 SCALE FACTOR CUT RATIO (FACTDN) IS  .66667,  SCALE FACTOR INCREASE RATIO (FACTUP) IS 1.20000
 SCALE FACTOR LOWER LIMIT (SLIMDN) IS  .00050,  SCALE FACTOR UPPER LIMIT (SLIMUP) IS  .80000
 SWITCH TO INCREASE MODE AT D CONVERGENCE  2.0000D-03 (SCRIT, RHF=INCR MODE ONLY).
0 CYCLE                   ENERGY                    SCF CONVERGENCE         CI CONVERGENCE         EXTRAPOLATION     SCALE      RESET
0 CI SCF       ELECTRONIC           TOTAL          COEFF      DENSITY      COEFF      DENSITY

   1   0      -54.057848203      -54.057848203             
   1   1      -54.057848203      -54.057848203                                       .0000D+00                     8.0000D-01    YES 
0AT TERMINATION TOTAL ENERGY IS      -54.057848  HARTREES          ORBITAL CONVERGENCE IS  .00000D+00

0S = 1.500 ( 1.500 EXPECTED),   S(S+1) =  3.7500 (  3.7500 EXPECTED)
1MOLECULAR ORBITALS         5 ALPHA SPIN ELECTRONS,   2 BETA SPIN ELECTRONS

                           1          2          3          4          5
 
    EIGENVALUES---   -15.49477    -.93164    -.27397    -.27397    -.27397
 
   1  1   N   1S        .99229    -.24422     .00000     .00000     .00000
   2  1   N   2S        .03464    1.02132     .00000     .00000     .00000
   3  1   N   2PX       .00000     .00000    1.00000     .00000     .00000
   4  1   N   2PY       .00000     .00000     .00000    1.00000     .00000
   5  1   N   2PZ       .00000     .00000     .00000     .00000    1.00000
 
 
 THE FIRST   2 MO ARE DOUBLY OCCUPIED FROZEN CORE ORBITALS
0OPEN SHELL ORBITAL OCCUPANCIES:    MO     OCCA     OCCB
                                     3    1.0000    .0000
                                     4    1.0000    .0000
                                     5    1.0000    .0000
1MULLIKEN POPULATION MATRIX CONDENSED TO ATOMS (ALPHA SPIN)     (DOUBLE THE OFF-DIAGONAL ELEMENTS FOR 'BOND ORDERS')

                           1
 
         1   N         5.00000
 
1MULLIKEN POPULATION MATRIX CONDENSED TO ATOMS (BETA SPIN)      (DOUBLE THE OFF-DIAGONAL ELEMENTS FOR 'BOND ORDERS')

                           1
 
         1   N         2.00000
 
1MULLIKEN POPULATION MATRIX CONDENSED TO ATOMS (TOTAL)          (DOUBLE THE OFF-DIAGONAL ELEMENTS FOR 'BOND ORDERS')

                           1
 
         1   N         7.00000
 
0ATOMIC CHARGES (ALPHA(1), BETA(2)), ATOMIC SPIN DENSITIES(3), ATOMIC CHARGES (TOTAL(4), NET(5))

                           1          2          3          4          5
 
         1   N         5.00000    2.00000    3.00000    7.00000     .00000
 
1BOND ORDER AND VALENCE INDICES (I. MAYER, I.J.Q.C., 29, 477-483 (1986))


          BOND ORDER MATRIX

                           1
 
         1   N          .00000
 
0                      VALENCE  FREE VALENCE

                           1          2
 
         1   N         3.00000    3.00000
 

0NORMAL TERMINATION, TOTAL CPU TIME FOR THIS CALCULATION WAS      .002 MINUTES.
1WELCOME TO U OF T MONSTERGAUSS                                                                         UNX VERSION: 13 AUG 1991
 ******************************
0CONTROL CARDS:

 *MOL BA=G,RHF                                                           

0SEQUENCE    GROUP     MG OPTIONS         GROUP OPTIONS (USER CONTROLLABLE)                  LINKS TO BE CALLED          JUMP

     1         2      000007012100 000000000000000000000000000000000000000000000000   01000000000000000000000000000000     0
     2         3      000007012100 004000010020000000000000000000000000000000000000   11000100000000000000000000000000     0
     3         4      000007012100 100000000000000010000000000000000000000000000000   11000000000000000000000000000000     0
     4         5      000007012100 100000000000000010000000000000000000000000000000   00100000000000000000000000000000     0
     5         6      000007012100 000000000000000000001000000000000000000000000000   10000000000000000000000000000000     0
0OPTION NUMBERS:      000000000111 111111122222222223333333333444444444455555555556   66666666677777777778888888888999
                      123456789012 345678901234567890123456789012345678901234567890   12345678901234567890123456789012

0CONVERSION FACTORS:  1 BOHR =  .52917706 ANGSTROMS
0                     1 HARTREE/BOHR = 8.2388584 MDYNE

                      1 HARTREE/ANGSTROM**2 = 4.3598149 MDYNE/ANGSTROM

                      1 ELECTRON*BOHR = 2.5417655 DEBYE

                      1 HARTREE/PARTICLE = 627.50959 KCAL/MOL = 2625.5001 KJ/MOL = 27.211611 EV/PARTICLE = 219474.67 CM(-1)/PARTICLE

                      1 ATOMIC UNIT OF ELECTRIC FIELD = 5.1422506D+09 VOLTS/CM
1OXYGEN                                                                  
 ------------------------------------------------------------------------
0CHARGE  =   0          MULTIPLICITY  =  3

0FREE FORMAT Z MATRIX:

  8                                                                      
                                                                         
0NO VARIABLES USED - SWITCHING TO STANDARD Z MATRIX FORMAT FOR OUTPUT
1
0---------------------------------------------------------------------------------------------------
                                             Z MATRIX
0  I   AN   Z1      BL                  Z2     ALPHA                Z3      BETA                Z4
 ---------------------------------------------------------------------------------------------------
   1    8
 ---------------------------------------------------------------------------------------------------


 ------------------------------------------------------------------------------------------------------------------------------------
                    COORDINATES IN ANGSTROMS                               COORDINATES IN BOHR                    ATOMIC ISOTOPE/MASS
   I   AN        X             Y             Z                        X             Y             Z               MASS#    MASS (AMU)
 ------------------------------------------------------------------------------------------------------------------------------------
   1    8     .0000000      .0000000      .0000000                 .0000000      .0000000      .0000000             16    15.99491464
 ------------------------------------------------------------------------------------------------------------------------------------

0PROJECTING FROM STO-3G BASIS (5D/7F)

0BASIS SET TYPE: EXTERNAL GENERAL     ( 5D/ 7F)
 POLARIZATION  : UNKNOWN                                                                              
 DIFFUSE FUNCS : UNKNOWN                                                                              
1BASIS SET FOR THIS CALCULATION (EXPONENTS ARE SCALED)

 **********************************************************************************************************************************
               ATOMIC ORBITAL                                     GAUSSIAN FUNCTIONS
 **********************************************************************************************************************************
      ATOM       NO.    TYPE   SCALE FACTOR    EXPONENT        S COEF         P COEF         D COEF         F COEF
 **********************************************************************************************************************************
       OXYGEN
                    1    1S       1.00000
                                             2.7755500D+02  7.0308000D-02   .0000000D+00   .0000000D+00   .0000000D+00
                                             4.1892300D+01  3.9585300D-01   .0000000D+00   .0000000D+00   .0000000D+00
                                             9.0443700D+00  6.6189400D-01   .0000000D+00   .0000000D+00   .0000000D+00
                    2    2S       1.00000
                                             9.2838300D+00 -8.7525800D-02   .0000000D+00   .0000000D+00   .0000000D+00
                                             1.1481200D+00  4.0767100D-01   .0000000D+00   .0000000D+00   .0000000D+00
                                             3.5383100D-01  6.7643200D-01   .0000000D+00   .0000000D+00   .0000000D+00
                3-  5    2P       1.00000
                                             8.0964800D+00   .0000000D+00  1.2332600D-01   .0000000D+00   .0000000D+00
                                             1.6776400D+00   .0000000D+00  4.7510100D-01   .0000000D+00   .0000000D+00
                                             3.7410100D-01   .0000000D+00  6.1517300D-01   .0000000D+00   .0000000D+00
 **********************************************************************************************************************************

0*** THE S=P EXPONENT CONSTRAINT HAS BEEN REMOVED
0RYSSET ROOT STATISTICS:     INTERPOLATED:       54     CALCULATED:        0     ASSYMPTOTIC APPROXIMATION:        0
0STANDARD CUTOFFS SELECTED IN SHELL.


         33 TWO ELECTRON INTEGRALS WRITTEN OUT ON UNIT  9 WITH INTEGRAL CUTOFF  1.00000D-10
1RHF OPEN SHELL SPECIFICATION
0GROUP     : FA   
 DEGENERACY: 23000
 ALPHA OCC : 23000
 BETA OCC  : 21000

0ORBITAL GROUP :       F       D       S       A       V
0NUMBER OF MO  :       2       0       0       3       0

0LEVEL    DEGENERACY    DOUBLE OCC    SINGLE ALPHA    SINGLE BETA    CONFIGURATIONS    SAMPLE CONFIGURATION
0  1           3             1              2               0               3          AB  A   A   
+                                                                                      __  __  __  

0COULOMB COUPLING COEFFICIENTS   A(I,J)
0  MO          OCCA      OCCB         CLOSED   OPEN 1   OPEN 2   OPEN 3   
0CLOSED       2.0000    2.0000        2.0000   1.3333   1.3333   1.3333
 OPEN 1       1.0000     .3333        1.3333    .6667    .8333    .8333
 OPEN 2       1.0000     .3333        1.3333    .8333    .6667    .8333
 OPEN 3       1.0000     .3333        1.3333    .8333    .8333    .6667

0EXCHANGE COUPLING COEFFICIENTS   B(I,J)
0  MO          OCCA      OCCB         CLOSED   OPEN 1   OPEN 2   OPEN 3   
0CLOSED       2.0000    2.0000        1.0000    .6667    .6667    .6667
 OPEN 1       1.0000     .3333         .6667    .3333    .5000    .5000
 OPEN 2       1.0000     .3333         .6667    .5000    .3333    .5000
 OPEN 3       1.0000     .3333         .6667    .5000    .5000    .3333

0PH INITIAL GUESS ... SCALE FACTOR (K/2) =  .8750
 SMALLEST EIGENVALUE OF OVERLAP IS  7.63296D-01
 SMALLEST EIGENVALUE OF OVERLAP IS  7.89245D-01
 COEFFICIENTS PROJECTED TO CURRENT BASIS
 
 THESE MO COEFFICIENTS HAVE BEEN WRITTEN TO FILE 25, RE-ORDERED BY GROUPS
 THESE MO COEFFICIENTS HAVE BEEN WRITTEN TO FILE 27, RE-ORDERED BY GROUPS
1RHF OPEN SHELL SCF               NUCLEAR REPULSION ENERGY IS       .000000000  HARTREES
0CONVERGENCE ON DENSITY MATRIX REQUIRED TO EXIT IS  1.0000D-06,  INITIAL SCALE FACTOR (SCALST) IS  .80000
 SCALE FACTOR CUT RATIO (FACTDN) IS  .66667,  SCALE FACTOR INCREASE RATIO (FACTUP) IS 1.20000
 SCALE FACTOR LOWER LIMIT (SLIMDN) IS  .00050,  SCALE FACTOR UPPER LIMIT (SLIMUP) IS  .80000
 SWITCH TO INCREASE MODE AT D CONVERGENCE  2.0000D-03 (SCRIT, RHF=INCR MODE ONLY).
0 CYCLE                   ENERGY                    SCF CONVERGENCE         CI CONVERGENCE         EXTRAPOLATION     SCALE      RESET
0 CI SCF       ELECTRONIC           TOTAL          COEFF      DENSITY      COEFF      DENSITY

   1   0      -74.334039457      -74.334039457             
   1   1      -74.334039457      -74.334039457                                       .0000D+00                     8.0000D-01    YES 
0AT TERMINATION TOTAL ENERGY IS      -74.334039  HARTREES          ORBITAL CONVERGENCE IS  .00000D+00

0S = 1.000 ( 1.000 EXPECTED),   S(S+1) =  2.0000 (  2.0000 EXPECTED)
1MOLECULAR ORBITALS         5 ALPHA SPIN ELECTRONS,   3 BETA SPIN ELECTRONS

                           1          2          3          4          5
 
    EIGENVALUES---   -20.49257   -1.22311    -.40008    -.40008    -.40008
 
   1  1   O   1S        .99191    -.25018     .00000     .00000     .00000
   2  1   O   2S        .03551    1.02236     .00000     .00000     .00000
   3  1   O   2PX       .00000     .00000    1.00000     .00000     .00000
   4  1   O   2PY       .00000     .00000     .00000    1.00000     .00000
   5  1   O   2PZ       .00000     .00000     .00000     .00000    1.00000
 
 
 THE FIRST   2 MO ARE DOUBLY OCCUPIED FROZEN CORE ORBITALS
0OPEN SHELL ORBITAL OCCUPANCIES:    MO     OCCA     OCCB
                                     3    1.0000    .3333
                                     4    1.0000    .3333
                                     5    1.0000    .3333
1MULLIKEN POPULATION MATRIX CONDENSED TO ATOMS (ALPHA SPIN)     (DOUBLE THE OFF-DIAGONAL ELEMENTS FOR 'BOND ORDERS')

                           1
 
         1   O         5.00000
 
1MULLIKEN POPULATION MATRIX CONDENSED TO ATOMS (BETA SPIN)      (DOUBLE THE OFF-DIAGONAL ELEMENTS FOR 'BOND ORDERS')

                           1
 
         1   O         3.00000
 
1MULLIKEN POPULATION MATRIX CONDENSED TO ATOMS (TOTAL)          (DOUBLE THE OFF-DIAGONAL ELEMENTS FOR 'BOND ORDERS')

                           1
 
         1   O         8.00000
 
0ATOMIC CHARGES (ALPHA(1), BETA(2)), ATOMIC SPIN DENSITIES(3), ATOMIC CHARGES (TOTAL(4), NET(5))

                           1          2          3          4          5
 
         1   O         5.00000    3.00000    2.00000    8.00000     .00000
 
1BOND ORDER AND VALENCE INDICES (I. MAYER, I.J.Q.C., 29, 477-483 (1986))


          BOND ORDER MATRIX

                           1
 
         1   O          .00000
 
0                      VALENCE  FREE VALENCE

                           1          2
 
         1   O         2.66667    2.66667
 

0NORMAL TERMINATION, TOTAL CPU TIME FOR THIS CALCULATION WAS      .002 MINUTES.
0.4u 0.2s 0:01 58%
